README.md

```diff
@@ -997,6 +1023,14 @@ From github.com:foo/bar
 (master)$ git mv --force myfile MyFile
 ```
 
+<a href="i-want-to-overwrite-local-files-when-doing-a-git-pull"></a>
+### I want to overwrite local files when doing a git pull.
+
+```sh
+$ git fetch --all
+$ git reset --hard origin/master
```

Comment:
Could you add the `(branch) $` to the example in this case? Are you on master when you do this? Would be nice to be explicit.